File: NewsUA.API/Models/News.cs
using System.ComponentModel.DataAnnotations;

namespace NewsUA.API.Models
{
    public class News
    {
        [Key]
        public int Id { get; set; }
        public string Title { get; set; }
        public string SubTitle { get; set; }
        public string AuthorName { get; set; }
        public string Information { get; set; }
        public string Status { get; set; }
        public bool IsHot { get; set; }
        public string? Type { get; set; }
        public DateTime? PublishedAt { get; set; }
        public DateTime? EdittedAt { get; set; }
    }
}

File: client/src/app/models/news.ts
export interface News {
    id: number
    title: string
    subTitle: string
    authorName: string
    information: string
    status: string
    isHot: boolean
    type: string | null
    publishedAt: Date
    edittedAt: Date | null
}

export interface EditNewsModel {
    id: number
    title: string
    subTitle: string
    authorName: string
    information: string
    status: string
    isHot: boolean
    type: string | null
}